apps/src/templates/sectionsRefresh/SingleSectionSetUp.jsx

```diff
@@ -30,8 +30,8 @@ export default function SingleSectionSetUp({
         name="grades"
         required={true}
         options={gradeOptions}
-        values={section.grades || []}
-        setValues={g => updateSection('grades', g)}
+        values={section.grade || []}
+        setValues={g => updateSection('grade', g)}
```

Comment:
was this breaking something? the api should support the new `grades` param for this

Reply:
It was, for the the "edit section" screen, it was looking at the data of an existing section.  The existing section had the param named "grade" not "grades".   I noticed this when testing the `SectionsSetupContainer` for the edit page. 

I do not have experience with the api you mentioned... should that also work for existing sections?